source/scss/_pf-basic-tags.scss

```diff
@@ -0,0 +1,17 @@
+//
+// Label
+// --------------------------------------------------
+
+.tag {
```

Comment:
All of these should be set on variables. ping me when you are online and I'll show you how, its super easy 😄